tests/getRevPagination.spec.js

```diff
@@ -24,9 +24,9 @@ describe("GET /api/reviews/:movieId", () => {
                 },
             })
         };
-
+        const movieId = 3;
         await getMovieReviews(3, cmsAdapterReviews)
-        request(app).get("/api/reviews/:movieId")
+        request(app).get(`/api/reviews/${movieId}`)
```

Comment:
Please change row 27 & 29 back to what it was before. The changes made no make the test no longer work.